backends/graphics3d/openglsdl/openglsdl-graphics3d.cpp

```diff
@@ -591,6 +617,12 @@ bool OpenGLSdlGraphics3dManager::createOrUpdateGLContext(uint gameWidth, uint ga
 		return false;
 
 	initializeOpenGLContext();
+#if defined(USE_IMGUI) && SDL_VERSION_ATLEAST(2, 0, 0)
+	_imguiInit = true;
```

Comment:
I wonder, should we init it always, or perhaps per request? E.g. at this moment, only one engine is using it, but we will unconditionally enable it for every engine.

It is not a blocker, thought